cpp/daal/src/algorithms/k_nearest_neighbors/oneapi/bf_knn_classification_predict_kernel_ucapi_impl.i

```diff
@@ -90,6 +90,10 @@ services::Status KNNClassificationPredictKernelUCAPI<algorithmFpType>::compute(c
     NumericTable * labels = const_cast<NumericTable *>(model->impl()->getLabels().get());
 
     const Parameter * const parameter = static_cast<const Parameter *>(par);
+    DAAL_CHECK(par, services::ErrorNullParameterNotSupported);
+    const bool computeLabels = bool(parameter->resultsToEvaluate & daal::algorithms::classifier::computeClassLabels) && (y != NULL);
+    const bool computeRegressionIndices = bool(parameter->resultsToEvaluate & ResultToComputeId::computeIndicesOfNeighbors) && (outIndices != NULL);
+    const bool computeRegressionDistances = bool(parameter->resultsToEvaluate & ResultToComputeId::computeDistances) && (outDistances != NULL);
```

Comment:
computeClassificationIndices / computeClassificationDistances?

Reply:
I suppose that it's not the best naming since it serves as a part of regression algorithm only

Reply:
You have "regression" name in classification algorithm, it confuses.
Also, I thought that we will expose indexes and distances in classification/search algos also, as we do this in CPU case.

Reply:
Wording is fixed